plugins/dualGovernance/hooks/useCanCreateProposal.tsx

```diff
@@ -56,29 +85,12 @@ export function useCanCreateProposal() {
     contractReads?.[2]?.status,
   ]);
 
-  // Check if PROPOSER_PERMISSION is granted to the current wallet
   useEffect(() => {
-    if (!address || !daoAddress) return;
+    setHasCreatePermission(!!hasCreatePermissionData);
+  }, [hasCreatePermissionStatus]);
```

Comment:
Why creating another effect just to do the same as before, but now with 2 layers?